pkg/kubectl/cmd/log.go

```diff
@@ -63,7 +63,7 @@ func selectContainer(pod *api.Pod, in io.Reader, out io.Writer) string {
 // NewCmdLog creates a new pod log command
 func NewCmdLog(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "log [-f] [-p] POD [CONTAINER]",
+		Use:     "logs [-f] [-p] POD [CONTAINER]",
```

Comment:
With command renames, we've been keeping the old one as an alias (with a deprecation warning) to avoid breaking existing uses. See [rollingupdate.go](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubectl/cmd/rollingupdate.go#L63) for an example that does this.